server/channels/app/platform/log.go

```diff
@@ -31,7 +32,7 @@ func (ps *PlatformService) initLogging() error {
 	// create the app logger if needed
 	if ps.logger == nil {
 		var err error
-		ps.logger, err = mlog.NewLogger()
+		ps.logger, err = mlog.NewLogger(logr.MaxFieldLen(*ps.Config().LogSettings.MaxFieldSize))
```

Comment:
Ideally, this should be a part of `mlog.LoggerConfiguration`, but I see that it's actually just a target configuration. I'd prefer if we change that to be a struct and add the `MaxFieldSize` there as well. But don't want to block the PR. Could be made as a future improvement.

Reply:
Added the ticket: https://mattermost.atlassian.net/browse/MM-54122